appveyor.yml

```diff
@@ -13,8 +13,8 @@ deploy:
   - provider: NuGet
     api_key:
       secure: MALWFhc9/aJSnKQhjY1OID0nl5LZpcWpP1vYAmTDQxhNkE41Y2Sft0zR1EOk6Nw3 
-    skip_symbols: false
-    artifact: /.*\.nupkg/
+    skip_symbols: true
```

Comment:
Is that right?  I also noticed there is a `symbol_server` field in the `appveyor.yml` reference, is that relevant?

Reply:
Aah yes! Looks like I had found a rather outdated workaround which used to be necessary before `.snupkg` support was added to appveyor. Looking at the appveyor docs I think `symbol_server` is what we want, using the url documented here:
https://docs.microsoft.com/en-us/nuget/api/symbol-package-publish-resource#push-a-symbol-package